integrator/integratorlib.py

```diff
@@ -0,0 +1,65 @@
+from subprocess import Popen
+
+
+def find_asteroid(i,f2,lrec,stlen):
```

Comment:
Please, use the good names of the variables:

- i -> asteroid_name
- f2 -> source_file
- lrec -> num_lines
- stlen -> ...